acceptance/managers/image_manager.go

```diff
@@ -0,0 +1,176 @@
+// +build acceptance
+
+package managers
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	dockertypes "github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+
+	h "github.com/buildpacks/pack/testhelpers"
+)
+
+var DefaultDuration = 10 * time.Second
+
+type ImageManager struct {
+	testObject *testing.T
+	assert     h.AssertionManager
+	dockerCli  client.CommonAPIClient
+}
+
+func NewImageManager(t *testing.T, dockerCli client.CommonAPIClient) ImageManager {
+	return ImageManager{
+		testObject: t,
+		assert:     h.NewAssertionManager(t),
+		dockerCli:  dockerCli,
+	}
+}
+
+func (im ImageManager) CleanupImages(imageNames ...string) {
+	im.testObject.Helper()
+	err := h.DockerRmi(im.dockerCli, imageNames...)
+	if err != nil {
+		im.testObject.Logf("%s: Failed to remove image from %s", err, imageNames)
+	}
```

Comment:
There are a number of times where we try to remove images that don't exist – for example, we do a `Cleanup` in the `it.After` for virtually all `pack build` acceptance tests, but our tests have some failure cases where a build is supposed to fail, and the resulting image wasn't created. As such, it didn't make sense to fail if we can't RM the image.